### PR TITLE
move explicit spec file info to existing envs file

### PR DESCRIPTION
**Pull request by electronwill**
move explicit spec file info to existing envs file, along with
conda env export and conda env create, to maintain cleaner navigation.

### CODE REVIEW

**Comment by electronwill** [comment]
Apologies for the second PR, but Kerry and I looked it over and decided the navigation would be cleaner this way. Content is unchanged. Ready to merge.

File: docs/source/using/envs.rst [hunk 1]
```diff
@@ -216,3 +216,38 @@ Verify that the new environment was installed correctly:
    conda list
 
 Next, we'll take a look at managing Python.
```
Comment (asmeurer) [inline docs/source/using/envs.rst]: Is this line still correct? I don't understand what it meant at the end of the file.

Comment (electronwill) [inline docs/source/using/envs.rst]: Oops! I'll fix that, one second

**Comment by electronwill** [comment]
OK, should be ready.

**Pull request merged**